.envrc

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/bash
+set -e
+
+if [ "$(python -V)" != "Python $(cat .python-version)" ]; then
```

Comment:
`.python-version` was only available locally on my old computer. It did not get added to the repo because git was ignoring.

Using Python 2.7 does not work with pdm, thus, needing at least Python 3.x version (I have not checked the minimum requirement. I'm assuming).